src/Microsoft.Health.Dicom.Core/Constants.cs

```diff
@@ -0,0 +1,14 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Core
+{
+    internal static class Constants
+    {
+        public const string OldestDeleteInstanceCacheKey = "_oldestDeleted";
```

Comment:
I think based on my other comments, you can move these comments to *BackgroundServiceHealthCheckCache.cs*